integration/quarkus/deployment/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>blaze-persistence-core-impl</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>blaze-persistence-entity-view-impl</artifactId>
+            <scope>compile</scope>
```

Comment:
Not that it's a big issue, but why do you need to access the implementation parts?